crates/compiler/parse/src/ast.rs

```diff
@@ -1135,6 +1138,8 @@ impl<'a> Pattern<'a> {
                     false
                 }
             }
+
+            Stmt(_) => todo!(),
```

Comment:
Is it intentional to still have these two `todo!`s here, or should they actually be handled?

Reply:
I missed them. I think they cam be unreachable or internal_error. I'll have a closer look.